spec/unit/plugins/aix/virtualization_spec.rb

```diff
@@ -20,11 +21,228 @@
+              PV_AU_WRITE,PV_AZ_ADMIN,PV_AZ_CHECK,PV_AZ_READ,PV_AZ_ROOT,
+              PV_DAC_,PV_DAC_GID,PV_DAC_O,PV_DAC_R,PV_DAC_RID,PV_DAC_UID,
+              PV_DAC_W,PV_DAC_X,PV_DEV_CONFIG,PV_DEV_QUERY,PV_FS_CHOWN,
+              PV_FS_CHROOT,PV_FS_CNTL,PV_FS_LINKDIR,PV_FS_MKNOD,
+              PV_FS_MOUNT,PV_FS_PDMODE,PV_FS_QUOTA,PV_KER_ACCT,
+              PV_KER_CONF,PV_KER_DR,PV_KER_EWLM,PV_KER_EXTCONF,
+              PV_KER_IPC,PV_KER_IPC_O,PV_KER_IPC_R,PV_KER_IPC_W,
+              PV_KER_NFS,PV_KER_RAC,PV_KER_RAS_ERR,PV_KER_REBOOT,
+              PV_NET_PORT,PV_PROC_CKPT,PV_PROC_CORE,PV_PROC_CRED,
+              PV_PROC_ENV,PV_PROC_PRIO,PV_PROC_PDMODE,PV_PROC_RAC,
+              PV_PROC_RTCLK,PV_PROC_SIG,PV_PROC_TIMER,PV_PROC_VARS,
+              PV_PROC_PRIV,PV_SU_UID,PV_TCB,PV_TP,PV_TP_SET,PV_MIC,
+              PV_MIC_CL,PV_LAB_,PV_LAB_CL,PV_LAB_CLTL,PV_LAB_LEF,
+              PV_LAB_SLDG,PV_LAB_SLDG_STR,PV_LAB_SL_FILE,PV_LAB_SL_PROC,
+              PV_LAB_SL_SELF,PV_LAB_SLUG,PV_LAB_SLUG_STR,PV_LAB_TL,
+              PV_MAC_,PV_MAC_CL,PV_MAC_R,PV_MAC_R_CL,PV_MAC_R_STR,
+              PV_MAC_R_PROC,PV_MAC_W,PV_MAC_W_CL,PV_MAC_W_DN,PV_MAC_W_UP,
+              PV_MAC_W_PROC,PV_MAC_OVRRD,PV_KER_SECCONFIG,
+              PV_PROBEVUE_TRC_USER,PV_PROBEVUE_TRC_USER_SELF,PV_KER_LVM
+
+DEVICE EXPORTS
+Name               Type     Virtual Device     RootVG   Status
+-----------------------------------------------------------------
+/dev/null          pseudo                               EXPORTED
+/dev/tty           pseudo                               EXPORTED
+/dev/console       pseudo                               EXPORTED
+/dev/zero          pseudo                               EXPORTED
+/dev/clone         pseudo                               EXPORTED
+/dev/sad           clone                                EXPORTED
+/dev/xti/tcp       clone                                EXPORTED
+/dev/xti/tcp6      clone                                EXPORTED
+/dev/xti/udp       clone                                EXPORTED
+/dev/xti/udp6      clone                                EXPORTED
+/dev/xti/unixdg    clone                                EXPORTED
+/dev/xti/unixst    clone                                EXPORTED
+/dev/error         pseudo                               EXPORTED
+/dev/errorctl      pseudo                               EXPORTED
+/dev/audit         pseudo                               EXPORTED
+/dev/nvram         pseudo                               EXPORTED
+
+
+LSWPAR_L
+
+    end
     let(:plugin) do
       p = get_plugin("aix/virtualization")
       allow(p).to receive(:collect_os).and_return(:aix)
       allow(p).to receive(:shell_out).with("uname -L").and_return(mock_shell_out(0, "29 l273pp027", nil))
       allow(p).to receive(:shell_out).with("uname -W").and_return(mock_shell_out(0, "0", nil))
+      allow(p).to receive(:shell_out).with("lswpar -L").and_return(mock_shell_out(0, @lswpar_L, nil))
       p.run
```

Comment:
Remove this line if you're going to use `plugin` elsewhere in the spec file. It doesn't make sense to run the plugin twice with different stubs.